### ISSUE TITLE
Not able to Add Access token

### ISSUE THREAD

**Issue by abhikum1** [jazz-community/rtc-git-connector#45]
Hello Team,

I am trying to connect git with RTC using your extension. After registering the GitLab repo, while entering my access token the screen is prompting me to again enter it. 

I have double cheked the setup of the dependencies and they seem to be working. Also, I have cross checked my access token and it seems to be correct. On opening the developer tools i see a request is failing (PFB) for screenshot.

![image](https://user-images.githubusercontent.com/53219459/61708012-62aa2580-ad69-11e9-85f7-e4d0862a9d37.png)

Looking forward for you reply.

Reagards,
Abhishek

**Comment by innerjoin** [comment]
@abhikum1 Thank you for your report. Can you confirm from looking at the requests in the developer tools that the call to `/tokenStore?key=` returns the correct GitLab access token?

@MartinBenninger sounds like the same issue I've experienced recently while using it. For me, from looking at the screenshot, it looks like as if the access token key and value are missing.

**Comment by abhikum1** [comment]
@innerjoin yes the correct key is returned: {"token":"uoho3ewxhPgbTk4ycxCe"}

Also, while looking at the console i noticed the browser blocked Cross Domain Request.

![image](https://user-images.githubusercontent.com/53219459/61711484-cdf7f580-ad71-11e9-8f34-1104579649a4.png)

However, i have allowed cross domain requests in the ccm application.
![image](https://user-images.githubusercontent.com/53219459/61711591-0566a200-ad72-11e9-9f31-cf97fadb83b6.png)

**Comment by innerjoin** [comment]
@abhikum1 Did you follow the the [3.3 White List Configuration](https://github.com/jazz-community/rtc-git-connector/wiki/3.3-White-List-Configuration) instructions from the documentation?

@MartinBenninger @SBI- Not sure whether the _Whitelist_ configuration is sufficient. If I remember correctly, it applies only when going through the `proxy` endpoint, which not seems to be the case here. What needs to be configured in addition for cross origin requests here?

**Comment by SBI-** [comment]
@abhikum1 Which version of gitlab are you using, and what kind of webserver / proxy-setup is being used? It looks like whatever reverse-proxy gitlab is behind is not handling any of the CORS headers.

**Comment by abhikum1** [comment]
Gitlab version : 11.10.8
we are using docker container hosted on RHEL 
https://docs.gitlab.com/omnibus/docker/

**Comment by SBI-** [comment]
That doesn't preclude an error on our side, by the way. It's just something I noticed in the example response.

**Comment by daytalker** [comment]
unfortunately I'm facing the same issue.
On two of our instances this setup is working, on two other instances it is not.
As far as I have checked every step twice, configurations are the same on all systems (deployment via ansible for everything).

Any further tips or hints why it is failing?
